eng/illink.targets

```diff
@@ -281,7 +281,8 @@
     <!-- When running from Desktop MSBuild, DOTNET_HOST_PATH is not set.
```

Comment:
Do we need to update this comment?
```suggestion
    <!-- When running from Desktop MSBuild and Visual Studio, DOTNET_HOST_PATH is not set.
```

Reply:
Visual Studio uses Desktop MSBuild so this comment should be fine.

Reply:
Thanks - just read https://github.com/dotnet/runtime/issues/42676#issuecomment-698411695.